lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -298,14 +292,34 @@ def add_framework_file_reference_to_native_target(native_target, pod_target, dep
           end
         end
 
-        # Sets the APPLICATION_EXTENSION_API_ONLY build setting to YES for all
-        # configurations of the given native target.
-        #
         def configure_app_extension_api_only_to_native_target(native_target)
           native_target.build_configurations.each do |config|
             config.build_settings['APPLICATION_EXTENSION_API_ONLY'] = 'YES'
           end
         end
+
+        def configure_schemes_for_pod_target(project, pod_target)
+          pod_target.specs.each do |spec|
+            scheme_name = spec.spec_type == :library ? pod_target.label : pod_target.non_library_spec_label(spec)
+            scheme_configuration = pod_target.scheme_for_spec(spec)
+            unless scheme_configuration.empty?
```

Comment:
nit: pull this block into its own function?

Reply:
what does this help with?

Reply:
just readability